src/modules/core/components/Comment/Comment.tsx

```diff
@@ -13,8 +13,8 @@ import HookedUserAvatar from '~users/HookedUserAvatar';
 import { AnyUser, Colony, useLoggedInUser } from '~data/index';
 import TextDecorator from '~lib/TextDecorator';
 
-import { userHasRole } from '../../../users/checks';
-import { getUserRolesForDomain } from '../../../transformers';
+import { userHasRole } from '~modules/users/checks';
```

Comment:
You should remove the empty line between `~lib` import and your 2 modules imports.
Please, do this for all the files. We follow the following logic generally:
- external libraries
- our modules that have aliases
- relative imports
Each of this import section is separated by an empty line. Additionally, within relative imports different levels are also separated by an empty line.

This way it will be really clean :)